src/testvideos.jl

```diff
@@ -36,15 +36,15 @@ const videofiles  =  Dict(
                                                 "Creative Commons: By Attribution 3.0 Unported (http://creativecommons.org/licenses/by/3.0/deed)",
                                                 "CC-BY NatureClip (http://youtube.com/natureclip)",
                                                 "http://downloadnatureclip.blogspot.com/p/download-links.html",
-                                                "https://archive.org/download/LadybirdOpeningWingsCCBYNatureClip/Ladybird%20opening%20wings%20CC-BY%20NatureClip.mp4",
+                                                "http://archive.org/download/LadybirdOpeningWingsCCBYNatureClip/Ladybird%20opening%20wings%20CC-BY%20NatureClip.mp4",
                                                 ),
 
                     "annie_oakley.ogg" => VideoFile("annie_oakley.ogg",
                                                     "The \"Little Sure Shot\" of the \"Wild West,\" exhibition of rifle shooting at glass balls.",
                                                     "pubic_domain (US)",
                                                     "",
                                                     "http://commons.wikimedia.org/wiki/File:Annie_Oakley_shooting_glass_balls,_1894.ogg",
-                                                    "https://upload.wikimedia.org/wikipedia/commons/8/87/Annie_Oakley_shooting_glass_balls%2C_1894.ogv"),
+                                                    "http://upload.wikimedia.org/wikipedia/commons/8/87/Annie_Oakley_shooting_glass_balls%2C_1894.ogv"),
```

Comment:
:(

Reply:
Same as above, for some reason my spotty internet connection (on a united flight) wasn't liking `download()`ing from https urls. I just changed them back